client/src/components/App.jsx

```diff
@@ -102,7 +96,10 @@ const App = () => {
         getStyles={getStyles}
         setCurrentProduct={setCurrentProduct}
       />
-      <RVC> <ReviewSorter /> <ReviewList id={id} reviews={reviews}/> </RVC>
+      <ReviewsContainer
+        id={id}
+        reviews={reviews}
+        isLoading={isLoading}/>
```

Comment:
unimportant right now. But it helps clarity wise to put the closing tag on a separate line break like
    isLoading={isLoading}
/>